### ISSUE TITLE
[enhancement] Row reorder for datatable

### ISSUE THREAD

**Issue by cod3rshotout** [sebastienheyd/boilerplate#69]
Would be useful to add the row reorder in the datatable like this one: https://editor.datatables.net/examples/extensions/rowReorder.html
So we can easily update the order of a record, what do you think?

**Comment by yasirjawed** [comment]
Yes we can do that, can you more information about this issue then i will continue.

**Comment by Stischu** [comment]
It would be greate if the row snaps in to place as you reorder the list, as in the example in the url above. It would also help if there were an option that the user can click anywhere on the row to drag.
